app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    short_command = command.split()[0]
+    source = command.split()[1]
+    path = command.split()[2]
+    if short_command != "mv":
+        return
+    if "/" not in path:
+        os.rename(source, path)
+        return
+    path_parts = path.split("/")[:-1]
+    filename = path.split("/")[-1]
+    path = ""
+    for part in path_parts:
+        path = os.path.join(path, part)
+    os.makedirs(path, exist_ok=True)
```

Comment:
You can use this to obtain a directory name `destination_directory = os.path.dirname(path)`. And then you can just make `os.makedirs(destination_directory, exist_ok=True)`